src/features/deeplinks/routing.ts

```diff
@@ -21,7 +21,7 @@ export const DEEPLINK_TO_SCREEN_CONFIGURATION: DeepLinksToScreenConfiguration =
       if (isTimestampExpired(parsedExpirationTimestamp)) {
         return {
           screen: 'ResetPasswordExpiredLink',
-          params: { email: params.email },
+          params: { email: decodeURIComponent(params.email) },
```

Comment:
Si tu mettre la logique de "parsing" au même endroit que `parsedExpirationTimestamp`, tu peux faire quelque chose du genre :
```suggestion
          const parsedEmail = decodeURIComponent(params.email)
          ...
          params: { email: parsedEmail }
```